src/content/docs/distribute/Signing/ios.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: iOS Code Signing
+sidebar:
+  label: iOS
+  order: 4
+---
+
+Code signing on iOS requires enrolling to the [Apple Developer] program.
```

Comment:
```suggestion
Code signing on iOS is required to distribute your application through the official [Apple App Store] or possibly alternative marketplaces in the European Union and in general to install and execute on end user devices.

## Prerequisites
Code signing on iOS requires enrolling to the [Apple Developer] program, which at the time of writing costs 99$ per year.
You also need an Apple device where you perform the code signing. This is required by the signing process and due to the [Apple TOS].
```

Reply:
Is this not in line with the intended flow?
I was thinking adding a one liner to explain why signing is neede for iOS would be helpful and having a dedicated prerequisites section.